randovania/game_connection/connector/dread_remote_connector.py

```diff
@@ -66,6 +107,21 @@ def new_player_location_received(self, state_or_region: str):
                                         None)
         self.PlayerLocationChanged.emit(PlayerLocationEvent(self.current_region, None))
 
+    def new_collected_locations_received(self, new_indices: bytes):
+        locations = set()
+        if new_indices.startswith(b"locations:"):
+            index = 0
+            for c in new_indices[len(b"locations:"):]:
```

Comment:
might be best to split `b'locations:'` into a variable to prevent using the same magic string by hand twice